server/main_chatbot.py

```
@@ -164,7 +167,9 @@ async def _ws_reject(websocket, text: str, *, close_reason: str = ""):
 
 def _warm_user_caches(user_id: str):
```

Ruff (0.15.10)

[warning] Missing return type annotation for private function `_warm_user_caches`

Add return type annotation: `None`

(ANN202)